applications/Unity.GrantManager/src/Unity.GrantManager.Web/Pages/Dashboard/Index.js

```diff
@@ -13,25 +13,31 @@
 
 
     unity.grantManager.dashboard.dashboard.getApplicationStatusCount().then(applicationStatus => {
-        initializeChart(applicationStatus.map(obj => obj.applicationStatus), applicationStatus.map(obj => obj.count), 
+        initializeChart(applicationStatus.map(obj => obj.applicationStatus), applicationStatus.map(obj => obj.count),
             'Application Status Overview', 'Total Submissions', 'APPLICATION STATUS OVERVIEW', "Count", 'applicationStatusChart')
     });
 
     unity.grantManager.dashboard.dashboard.getApplicationTagsCount().then(applicationTags => {
-        initializeChart(applicationTags.map(obj => obj.applicationTag), applicationTags.map(obj => obj.count), 
+        initializeChart(applicationTags.map(obj => obj.applicationTag), applicationTags.map(obj => obj.count),
             'Application Tags Overview', 'Total Number of Tags', 'APPLICATION TAGS OVERVIEW', "Count", 'applicationTagsChart')
     });
 
+    let colorPalette;
+
+    fetch('./colorsPalette.json')
```

Comment:
Don't think we need to extract this to another file, hardcoding into the colors should be fine, I don't see us deploying just the colorsPalette on its own.